packages/ui/src/components/ColumnManager.tsx

```diff
@@ -28,29 +24,17 @@ function ColumnMenuItem<RowDataModel>({
     (typeof column.columnDef.header === 'string' ? column.columnDef.header : column.id);
 
   return (
-    <Dropdown.Item
```

Comment:
Using dropdown item showed a hover style on each row, and a primary blue background when a row was active (mouse button down). This was unnecessary as the whole row is not interactable.